nbbuild/cluster.properties

```diff
@@ -763,6 +762,7 @@ nb.cluster.enterprise=\
         j2ee.sun.ddui,\
         j2eeapis,\
         j2eeserver,\
+        jakarta.web.beans,\
```

Comment:
Why did you remove `languages.hcl`?

Reply:
I'm did not intentionally remove it.  Thanks for the review.  I will make the adjustments to add back.